nimble/controller/src/ble_ll_sched.c

```diff
@@ -892,6 +892,83 @@ ble_ll_sched_adv_new(struct ble_ll_sched_item *sch, ble_ll_sched_adv_new_cb cb,
     return 0;
 }
 
+int
+ble_ll_sched_periodic_adv(struct ble_ll_sched_item *sch, uint32_t *start,
+                          bool after_overlap)
+{
+    int rc;
+    os_sr_t sr;
+    uint32_t adv_start;
+    uint32_t duration;
+    struct ble_ll_sched_item *entry;
+#ifdef BLE_XCVR_RFCLK
+    struct ble_ll_sched_item *orig = sch;
+#endif
+
+    /* Get length of schedule item */
+    duration = sch->end_time - sch->start_time;
+
+    OS_ENTER_CRITICAL(sr);
+    entry = ble_ll_sched_insert_if_empty(sch);
+    if (!entry) {
+        rc = 0;
```

Comment:
seems like this function was c&p before https://github.com/apache/mynewt-nimble/pull/357/commits/45565e10f70f63b7bc7f6b9fb1cca22493d1e542 was merged :)

Reply:
no,  this is used to schedule all periodic events and  we want to fail (and skip event) if after_overlap is false and we overlap